WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -2921,7 +2922,12 @@ private void handleReblogStateChanges() {
                     );
                 } else if (state instanceof PostEditor) {
                     PostEditor peState = (PostEditor) state;
-                    ActivityLauncher.openEditorForReblog(getActivity(), peState.getSite(), peState.getPost());
```

Comment:
I'd rename the `peState` to something better :-)

Reply:
tbh was pretty in doubt on how to rename it without changing the variable semantic 😅...ended up with `postEditorStateData` 😬.

Reply:
sounds good :D it's just that I haven't seen these single letter variables since the times of C++